test/cql-pytest/test_select_mutation_fragments.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests concerning the SELECT MUTATION FRAGMENT statement, which allows dumping
```

Comment:
Don't forget to change the references here (and in commit messages and perhaps other places) to the new syntax.